test/integration/features/basic.feature

```diff
@@ -21,6 +21,17 @@ Feature: Basic
      sudoer
      """
 
+  Scenario: Current Context should be minishift
```

Comment:
@hferentschik Agree, let me extend it. was thinking something different which didn't make sense but this does.